### PR TITLE
Removed some free features for new users

### PR DESCRIPTION
**Pull request by girishpanchal30**
I've disabled some free features for new users only, now premium users can use those features.

Close Codeinwp/tweet-old-post-pro/issues/523

### CODE REVIEW

File: vue/src/vue-elements/post-format.vue [hunk 1]
```diff
@@ -764,7 +817,10 @@
             },
             shorteners: function () {
                 return this.$store.state.generalSettings.available_shorteners;
-            }
+            },
+            isNewUserPro: function () {
+                return 0 === this.license && this.$store.state.is_new_user;
```
[the remark this answers is not in the document]
Comment (girishpanchal30) [reply vue/src/vue-elements/post-format.vue:823]: @Soare-Robert-Daniel Done!, please review with my latest commit.

File: vue/src/vue-elements/post-format.vue [hunk 2]
```diff
@@ -819,6 +819,7 @@
                 return this.$store.state.generalSettings.available_shorteners;
             },
             isNewUserPro: function () {
+                console.log(0 === this.license && this.$store.state.is_new_user);
```
Comment (Soare-Robert-Daniel) [inline vue/src/vue-elements/post-format.vue:822]: You can remove the `console.log`. 

If you doubt the data type, you can wrap the response in `Boolean( ... )`.

**Review by Soare-Robert-Daniel — CHANGES_REQUESTED** [review]
Just remove that missed `console.log`. The rest is good 👌 

Regarding the failing E2E, the problem is the old packages. They are updated in https://github.com/Codeinwp/tweet-old-post/pull/974 -- As a context on why they are failing.

File: vue/src/vue-elements/post-format.vue (hunk 2 above)
Comment (girishpanchal30) [reply vue/src/vue-elements/post-format.vue]: @Soare-Robert-Daniel Done! Thank you for the review

**Review by Soare-Robert-Daniel — APPROVED** [review]

**Comment by girishpanchal30** [comment]
@rodica-andronache I've used this option to identify new users.
https://github.com/Codeinwp/tweet-old-post/pull/975/commits/73729499620b26880266bb68e5d442a55e61495f#diff-fcd6c0c59094f9f01bad1a79875057bfc1becc743535d2cdb5c90f7e2bedd202R392

**Comment by girishpanchal30** [comment]
@rodica-andronache @selul I've added a new option to identify new users based on the last install time. Please recheck with the latest build zip.

**Comment by rodica-andronache** [comment]
@girishpanchal30 I've tried again, on a fresh instance but it doesn't work for me. The options are still there, in the free version. What else should I try?

**Comment by girishpanchal30** [comment]
@rodica-andronache Can you please share your test instance details here?

**Comment by rodica-andronache** [comment]
@girishpanchal30 
`
https://glowing-stoat-d8ecc5.vertisite.cloud/wp-admin
user: rodica
pass: g0L@K8ATFM)qXFOErEsJ6UgG
`

**Comment by girishpanchal30** [comment]
@rodica-andronache It seems issue with the -1 license, now it is fixed, please check with the latest commit.

**Comment by rodica-andronache** [comment]
@girishpanchal30
On a fresh site ( so this is a new user ) everything worked well. There are just two things to mention:
- for these two upsells https://vertis.d.pr/i/VNtxtY I think the text should be "Custom meta field**s** are only available in the Pro version" and "Custom field**s** are only available in the Pro version"
- here, from what I understood from the issue, reviv.ly should be the default option. Right now, the field is empty by default https://vertis.d.pr/i/EO4M9k

To test what happens to an older user, I had ROP on a site, installed for more than 1h, I deleted it and installed this new version. But the features are still missing, like this would be a new user as well. Is there something I'm missing?
Here is the instance:

```
https://rop-test.vertisite.cloud/wp-admin/
user: rodica
password: sfE@sWP(Iyd7WVK$Fifwti*k
```

**Comment by girishpanchal30** [comment]
@rodica-andronache Fixed all 3 mentioned issues with the latest commit, please recheck with the newest build zip.

**Comment by rodica-andronache** [comment]
@girishpanchal30 the first two issues are fixed. But the third one is still happening for me. Let me know if there's something I can do on my side

**Comment by selul** [comment]
@rodica-andronache can you follow the next testing case on a Fresh instance: 

Install current version of rop from wporg. Update install time to 1 day before. Download the zip from here and install it. Check if the features are preserved.

**Comment by rodica-andronache** [comment]
@girishpanchal30 it works now, thank you!

**Pull request merged**